.ghci

```diff
@@ -19,3 +19,4 @@
 
 :set -XFlexibleInstances
 :set -XRankNTypes
+:set -XStandaloneDeriving
```

Comment:
Let's you do this: https://github.com/thoughtbot/carnival/pull/200/files#diff-dbe3d0b943e814dec632bd166fbce775R9